assets/blocks/algaeRoof.block

```diff
@@ -0,0 +1,6 @@
+{
+    "displayName": "algae Roof",
```

Comment:
If you don't provide additional information or properties in the `.block` files you could also omit them and just place the texture under `assets/blockTiles/auto/algaeRoof.png`.